src/main/scala/vexriscv/plugin/CsrPlugin.scala

```diff
@@ -649,7 +654,7 @@ class CsrPlugin(val config: CsrPluginConfig) extends Plugin[VexRiscv] with Excep
     pipeline.update(MPP, UInt(2 bits))
 
     if(withExternalMhartid) externalMhartId = in UInt(mhartidWidth bits)
-    if(utimeAccess != CsrAccess.NONE) utime = in UInt(64 bits) setName("utime")
+    if(pipeline.serviceExist(classOf[CounterService]) && utimeAccess != CsrAccess.NONE) utime = in UInt(64 bits) setName("utime")
```

Comment:
Ahhh this seems broken, shouldn't it be  !pipeline.serviceExist(classOf[CounterService]) ?

Reply:
Yes, this must have slipped by.